applications/adv_networking_bench/cpp/CMakeLists.txt

```diff
@@ -104,43 +104,55 @@ install(
 
 # Add testing
 if(BUILD_TESTING)
-
-  ## adv_networking_bench_default_tx_rx_test ##
-  # The test below is expected to run on an IGX devkit with a loopback
-  # connecting both ports of the IGX NIC (default config currently hardcodes
-  # the BUS address of these two ports), with optimal system setups (BAR1,
+  ## adv_networking_bench_dpdk_loopback_test ##
```

Comment:
On the same topic as before, "loopback" is an overloaded term and in the future we may want multiple versions. IE true loopback with a cable (this config), hardware loopback without a cable (TBD), and software loopback (the other PR)

Reply:
Renamed: https://github.com/nvidia-holoscan/holohub/compare/f03f9e9496f4dd5bbdf87c5e395e6607e094d46e..d94f65840789c265a83fe5b22768eb2cc7890a3d